ui: back links and heading spacing on details page

vladlos opened the pull request:
Used `Button` component, updated font and icon, clean up code

Resolves: #45446, #45447

Release note (ui): none

vladlos committed:
ui: back links and heading spacing on details page

Used `Button` component, updated font and icon, clean up code

Resolves: #45446

Release note (ui): none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/45588)
<!-- Reviewable:end -->

vladlos committed:
ui: Node details back button

Added back button to `Node Details` page

Resolves: #45447

Release note (ui): none

nathanstilwell reviewed (approved):
LGTM
